packages/config/schema.json

```
@@ -50,7 +50,26 @@
       "default": []
     },
     "commit": {
-      "type": "boolean",
+      "anyOf": [
+        {
+          "type": "array",
+          "items": [
+            {
+              "type": "string",
+              "title": "Commit Generator Path"
+            },
+            { "title": "Commit Generator Options" }
+          ],
+          "minItems": 2
```

I think that this should be:
```suggestion
          "minItems": 1,
          "maxItems": 2,
```

The same change could be done to the `changelog` option.